buzzmobile/tests/__init__.py

```diff
@@ -0,0 +1 @@
+import tests.rostest_utils
```

Comment:
have you checked if this is needed? I don't particularly care, and it won't block this PR but I'd like to know